torch_xla/csrc/init_python_bindings.cpp

```diff
@@ -852,6 +852,7 @@ void InitXlaModuleBindings(py::module m) {
         [](const at::Tensor& tensor) { return GetTensorViewAliasId(tensor); });
   m.def("_xla_get_tensor_id",
         [](const at::Tensor& tensor) { return GetTensorId(tensor); });
+  m.def("_xla_get_lock_spmd_config", []() { return lock_spmd_config; });
```

Comment:
This syntax seems wrong?

Reply:
Hm,.. can't see what you are seeing. Could you help me find the bug here, and I will test it again?

Reply:
Agreed, let me make that change.

Reply:
Done, thanks @alanwaketan for the suggestion :)